pgsmo/objects/view/view_templates/pg/9.1_plus/properties.sql

```diff
@@ -26,7 +26,7 @@ SELECT
         pg_seclabels sl1
      WHERE
         sl1.objoid=c.oid AND sl1.objsubid=0
-    ) AS seclabels,
+    ) AS seclabels
```

Comment:
Was this causing any issues?

Reply:
Yes, the query would give an error "ERROR: syntax error at or near "FROM" at character 620"

Reply:
Ok let's mention syntax error in PR title and description, before merging